lib/base/utility.cpp

```diff
@@ -1946,3 +1938,20 @@ String Utility::GetIcingaDataPath(void)
 }
 
 #endif /* _WIN32 */
+
+void Utility::RenameFile(const String& source, const String& destination)
+{
+#ifdef _WIN32
+	if (remove(destination.CStr()) && errno != ENOENT)
```

Comment:
@dnsmichi
1. `remove()` is standard-C++ – `_unlink()` isn't. (But they do the same.)
2. No, I didn't. (But `posix_exception` was already being thrown before I touched the code so I didn't *add* any problems.)